modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/succubus_milk.dm

```diff
@@ -80,112 +80,78 @@
-			path = new /obj/item/organ/external/genital/breasts
-			path.build_from_dna(exposed_mob.dna, ORGAN_SLOT_BREASTS)
-			path.Insert(exposed_mob, FALSE, FALSE)
-			var/obj/item/organ/external/genital/new_breasts = exposed_mob.getorganslot(ORGAN_SLOT_BREASTS)
-			new_breasts.genital_size = 2
-			new_breasts.update_sprite_suffix()
-			exposed_mob.update_body()
-			enlargement_amount = 0
-			if(new_breasts.visibility_preference == GENITAL_ALWAYS_SHOW || exposed_mob.is_topless())
-				exposed_mob.visible_message(span_notice("[exposed_mob]'s bust suddenly expands!"))
-				to_chat(exposed_mob, span_purple("Your chest feels warm, tingling with sensitivity as it expands outward."))
+	if(mob_breasts.visibility_preference == GENITAL_ALWAYS_SHOW || exposed_mob.is_topless())
+		switch(translation)
+			if(BREAST_SIZE_FLATCHESTED)
 				return
+			if(BREAST_SIZE_BEYOND_MEASUREMENT)
+				exposed_mob.visible_message(span_notice("[exposed_mob]'s [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(public_bigger_action_text_list)]"))
+				to_chat(exposed_mob, span_purple("Your [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(action_text_list)]about [mob_breasts.genital_size] inches in diameter."))
 			else
-				exposed_mob.visible_message(span_notice("The area around [exposed_mob]'s chest suddenly bounces a bit."))
-				to_chat(exposed_mob, span_purple("Your chest feels warm, tingling with sensitivity as it strains against your clothes."))
-				return
-	// Separates gender change stuff from breast growth.
-	if(exposed_mob.client?.prefs.read_preference(/datum/preference/toggle/erp/gender_change))
-		var/obj/item/organ/external/genital/penis/mob_penis = exposed_mob.getorganslot(ORGAN_SLOT_PENIS)
-		var/obj/item/organ/external/genital/testicles/mob_testicles = exposed_mob.getorganslot(ORGAN_SLOT_TESTICLES)
-		if(exposed_mob.gender == MALE)
-			exposed_mob.set_gender(FEMALE)
-			exposed_mob.physique = exposed_mob.gender
-			exposed_mob.update_body()
-			exposed_mob.update_mutations_overlay()
-		if(!mob_penis)
-			return
-		if(exposed_mob.client?.prefs.read_preference(/datum/preference/toggle/erp/penis_enlargement)) // To do cock shrinkage, check if prefs allow for this.
-			if(mob_penis.genital_size > penis_min_length)
-				mob_penis.genital_size -= penis_size_reduction_step
-				mob_penis.update_sprite_suffix()
-				exposed_mob.update_body()
-			if(mob_penis.girth > penis_minimum_girth)
-				mob_penis.girth -= penis_girth_reduction_step
-				mob_penis.update_sprite_suffix()
-				exposed_mob.update_body()
-			if(!mob_testicles || mob_testicles.genital_size <= 1)
+				if(mob_breasts?.genital_size >= (max_breast_size - 2))
+					exposed_mob.visible_message(span_notice("[exposed_mob]'s [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(public_bigger_action_text_list)]"))
+					to_chat(exposed_mob, span_purple("Your [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(action_text_list)]about [translation]-cups."))
```

Comment:
These messages could be stored in variables instead, and the proc calls could be done afterwards, no? That might be a little cleaner, I feel like. It would make those lines a little shorter, at least, for sure.